src/apollo/client-state/web3Resolvers.js

```diff
@@ -20,14 +20,15 @@ export const web3Resolvers = {
           let provider
           try {
             provider = await getWriteProvider()
+          } catch (e) {}
+          if (!provider) { return null }
```

Comment:
Indentation may be wrong here?

Reply:
Indentation is right; it's just janky code.  We need the empty catch at the moment.